metrics/metrics.go

```diff
@@ -53,9 +53,12 @@ func ParseMetricsKind(t string) Kind {
 // Metrics is the generic interface that all the required backends
 // should implement to be an skipper metrics compatible backend.
 type Metrics interface {
+	// Implements the `filter.Metrics` interface.
```

Comment:
Then you can also use filters.Metrics to pull the function receiver signatures into this interface.